docs/examples/llm/vllm.ipynb

```diff
@@ -0,0 +1,284 @@
+{
```

Comment:
<p>does it require to install ray for running vllm locally</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>